src/NumberTheory/GaloisGrp/GaloisGrp.jl

```diff
@@ -2664,7 +2664,7 @@ function blow_up(G::PermGroup, C::GaloisCtx, lf::Vector, con::PermGroupElem=one(
 
   icon = inv(con)
 
-  gs = map(Vector, gens(G))
+  gs = map(Vector{Int}, gens(G))
```

Comment:
```suggestion
  gs = Vector{Int}.(gens(G))
```
I am pretty sure the current form won't work, as mapping over an empty list and broadcasting over an empty list infer differently.
In case the tests fail, you could try to adapt the suggestion

Reply:
Mapping like that works and produces type stable code if the "function" being applied is a concrete type.